KaiUIngInferno/src/ui/Button.tsx

```diff
@@ -34,7 +43,15 @@ export default function Button({ text, isFocused, onClick, icon, iconSrc, iconSi
   );
 
   return (
-    <div className={buttonCls}>
+    <div onKeyDown={ linkEvent(props, handleKeyDown) } className={buttonCls} tabIndex={0} ref={(ref: HTMLElement | null) => {
+      if (ref) {
+        if (isFocused) {
+          ref.focus();
+        } else {
+          ref.blur();
+        }
+      }
```

Comment:
I'm not sure when this is run? is it run when the component first enters the DOM? or is it run on every render call? I would prefer setting a class variable to the ref and running the is focused stuff in the correct inferno lifecycle function (I guess it's the one when props change?)

Reply:
You are right it's not good for performance: https://github.com/infernojs/inferno/issues/1454